src/main/resources/db/migration/V1_95_4__referral_details_backfill_with_existing_referrals.sql

```diff
@@ -0,0 +1,26 @@
+insert into referral_details (
+    id,
+    superseded_by_id,
+    created_at,
+    created_by,
+    reason_for_change,
+    referral_id,
+    completion_deadline,
+    further_information,
+    maximum_enforceable_days
+)
+select
+    gen_random_uuid(),
+    null,
+    ref.created_at,
+    ref.created_by_id,
+    'initial referral details',
```

Comment:
should we make this field have some indication that these came from existing referrals?


it's content has no impact on anything really.